docs/website/docs/dlt-ecosystem/verified-sources/hubspot.md

```diff
@@ -172,7 +173,7 @@ def companies(
       api_key: str = dlt.secrets.value, include_history: bool = False
```

Comment:
could you update this resource description? add new arguments "props" and "include_custom_props"

Reply:
Oh, yeah, done